src/analysis/trampoline_parameters.rs

```diff
@@ -35,8 +35,8 @@ impl CParameter {
 
 #[derive(Clone, Debug)]
 pub struct Transformation {
-    pub ind_c: usize,    //index in `Vec<CParameter>`
-    pub ind_rust: usize, //index in `Vec<RustParameter>`
+    pub ind_c: usize,    // index in `Vec<CParameter>`
+    pub ind_rust: usize, // index in `Vec<RustParameter>`
```

Comment:
Make this doc comments? Maybe with intradoc link?

Reply:
I simply added a whitespace, we have equivalents a bit everywhere. Don't hesitate to send a PR to turn them into doc comments. :)